src/lib.rs

```diff
@@ -183,6 +185,13 @@ impl Identity {
         let identity = imp::Identity::from_pkcs12(der, password)?;
         Ok(Identity(identity))
     }
+
+    /// buf is the contents of a file containing a chain of PEM encoded certificates
```

Comment:
* Don't call the argument `buf`
* Specify the order in which we expect the certificates
* Specify that you expect concatenation